libraries.json

```diff
@@ -851,7 +851,7 @@
     "language": "C#"
   },
   "Yii": {
-    "imports": ["yii\\", "yiisoft/yii"],
+    "imports": ["yii\\", "yiisoft/yii", "yiisoft/yii2"],
```

Comment:
I think it would make sense to add Yii2 as a new library instead of merging with Yii V1. As I know these two are quite different.

Reply:
Agree